code/game/gamemodes/wizard/rightandwrong.dm

```diff
@@ -32,66 +32,90 @@
+					G = new /obj/item/weapon/gun/energy/lasercannon(get_turf(H))
 				if("crossbow")
-					var/obj/item/weapon/gun/energy/kinetic_accelerator/crossbow/large/gat  = new(get_turf(H))
-					gat.pin = new /obj/item/device/firing_pin
+					G = new /obj/item/weapon/gun/energy/kinetic_accelerator/crossbow/large(get_turf(H))
 				if("nuclear")
-					var/obj/item/weapon/gun/energy/gun/nuclear/gat  = new(get_turf(H))
-					gat.pin = new /obj/item/device/firing_pin
+					G = new /obj/item/weapon/gun/energy/gun/nuclear(get_turf(H))
 				if("sabr")
-					var/obj/item/weapon/gun/projectile/automatic/proto/gat  = new(get_turf(H))
-					gat.pin = new /obj/item/device/firing_pin
+					G = new /obj/item/weapon/gun/projectile/automatic/proto(get_turf(H))
 				if("bulldog")
-					var/obj/item/weapon/gun/projectile/automatic/shotgun/bulldog/gat  = new(get_turf(H))
-					gat.pin = new /obj/item/device/firing_pin
+					G = new /obj/item/weapon/gun/projectile/automatic/shotgun/bulldog(get_turf(H))
 				if("c20r")
-					var/obj/item/weapon/gun/projectile/automatic/c20r/gat = new(get_turf(H))
-					gat.pin = new /obj/item/device/firing_pin
+					G = new /obj/item/weapon/gun/projectile/automatic/c20r(get_turf(H))
 				if("saw")
-					var/obj/item/weapon/gun/projectile/automatic/l6_saw/gat  = new(get_turf(H))
-					gat.pin = new /obj/item/device/firing_pin
+					G = new /obj/item/weapon/gun/projectile/automatic/l6_saw(get_turf(H))
 				if("car")
-					var/obj/item/weapon/gun/projectile/automatic/m90/gat  = new(get_turf(H))
-					gat.pin = new /obj/item/device/firing_pin
+					G = new /obj/item/weapon/gun/projectile/automatic/m90(get_turf(H))
+				if("alienpistol")
+					G = new /obj/item/weapon/gun/energy/alien(get_turf(H))
+				if("dragnet")
+					G = new /obj/item/weapon/gun/energy/gun/dragnet(get_turf(H))
+				if("turret")
+					G = new /obj/item/weapon/gun/energy/gun/turret(get_turf(H))
+				if("pulsecarbine")
+					G = new /obj/item/weapon/gun/energy/pulse/carbine(get_turf(H))
+				if("decloner")
+					G = new /obj/item/weapon/gun/energy/decloner(get_turf(H))
+				if("mindflayer")
+					G = new /obj/item/weapon/gun/energy/mindflayer(get_turf(H))
+				if("hyperkinetic")
+					G = new /obj/item/weapon/gun/energy/kinetic_accelerator/hyper(get_turf(H))
+				if("advplasmacutter")
+					G = new /obj/item/weapon/gun/energy/plasmacutter/adv(get_turf(H))
+				if("wormhole")
+					G = new /obj/item/weapon/gun/energy/wormhole_projector(get_turf(H))
+				if("wt550")
+					G = new /obj/item/weapon/gun/projectile/automatic/wt550(get_turf(H))
+				if("bulldog")
+					G = new /obj/item/weapon/gun/projectile/automatic/shotgun(get_turf(H))
+				if("grenadelauncher")
+					G = new /obj/item/weapon/gun/projectile/revolver/grenadelauncher(get_turf(H))
+				if("goldenrevolver")
+					G = new /obj/item/weapon/gun/projectile/revolver/golden(get_turf(H))
+				if("sniper")
+					G = new /obj/item/weapon/gun/projectile/sniper_rifle(get_turf(H))
+				if("medibeam")

```

Comment:
Actually if you wrap something before sending it to cargo, Disposals will reject it from the trash and put it in the mail slot.